src/script.ts

```diff
@@ -30,7 +32,7 @@ export interface ScriptConfig extends ScriptReference {
    * An undefined command is valid as a way to give name to a group of other
    * scripts (specified as dependencies).
    */
-  command: string | undefined;
+  command: AstNode<string> | undefined;
```

Comment:
Threading the `AstNodes` through into the resolved script configs implies that it will be useful to have access to them in the `Executor`.

I think that _most_ of the errors for which it will be highly useful to have JSON squigglies would be caught in the `Analyzer`, since those are the errors that always require you to update the wireit config.

In the `Executor`, it seems maybe slightly less useful to show JSON squigglies, because most of the errors you encounter there are errors from the underlying script processes, rather than the build config, and usually the fix is not related to the wireit config. I could still seem some utility though, like if the error was because my command was wrong, this would let me quickly jump to it. Is that the kind of thing you were thinking?

Reply:
My thinking is that, to provide editor features we'll want down the line, we'll want to keep the relevant ASTNodes around on the ScriptConfig objects. I agree that _probably_ the executor won't have much use for them, but the typings are good enough that I don't think it'll be a problem

Reply:
Sgtm